sdk/core/core-http/lib/policies/serviceBusAtomSerializationPolicy.ts

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { HttpOperationResponse } from "../httpOperationResponse";
+import { WebResource } from "../webResource";
+import {
+  BaseRequestPolicy,
+  RequestPolicy,
+  RequestPolicyFactory,
+  RequestPolicyOptions
+} from "./requestPolicy";
+import { Constants } from "../util/constants";
+import { parseAtomXML, parseStringError } from "../util/xml";
+import { Buffer } from "buffer";
+import { ResourceSerializer } from "../resourceSerializer";
+const _ = require("underscore");
+
+/**
+ * Create a new serialization RequestPolicyCreator that will serialize/deserialize
+ * HTTP request bodies as they pass through the HTTP pipeline.
+ */
+export function serviceBusAtomSerializationPolicy(): RequestPolicyFactory {
+  return {
+    create: (nextPolicy: RequestPolicy, options: RequestPolicyOptions) => {
+      return new ServiceBusAtomSerializationPolicy(nextPolicy, options);
+    }
+  };
+}
+
+/**
+ * A RequestPolicy that will
+ *  - serialize HTTP requests with input in JSON to ATOM based XML requests, and
+ *  - deserialize the ATOM based XML responses as they pass through the HTTP pipeline.
+ */
+export class ServiceBusAtomSerializationPolicy extends BaseRequestPolicy {
```

Comment:
Does this policy make any assumptions on how Azure Service Bus expects requests and returns responses? If yes, then this should move to the Service Bus library. If this policy is service agnostic, then it can stay in core-http